frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -138,12 +145,18 @@ describe('Cluster Settings page', () => {
 describe('Cluster Version Details Table page', () => {
   let wrapper: ShallowWrapper<any>;
   let cv;
+  let storeSpy;
 
   beforeEach(() => {
+    storeSpy = jest.spyOn(storeHandler, 'getStore').mockImplementation(() => store);
```

Comment:
do we need to spy `storeHandler` or the hooks/utils that this component uses underlying? i.e `useAccessReview` we can spy on for instance

Below would fix issues with describe block `describe('Cluster Version Details Table page' ...`

```
 const spyUseAccessReview = jest.spyOn(utils, 'useAccessReview'); // spy on utils depending on the usage 
 ...
 spyUseAccessReview.mockReturnValue(true); // mock depending on the need
```

This would apply for other test files as well

Reply:
Pushed the update, the only thing that should be missing is the resolution of your comment around exposing stuff from the API and `useAccesReview` hooks waiting for Bryan's or Andrew's opinion

Reply:
Good point, it should be fixed now. Thanks @invincibleJai